From: Jochen Sprickerhof <git@jochen.sprickerhof.de>
Date: Thu, 24 Dec 2015 14:32:15 +0100
Subject: Don't set RPATH on tf2_py Python library (Debian specific)

Somehow cmake loses the tf2_py file when removing RPATHs upon
installation. We assume that files are installed into their default
location and don't need an RPATH accordingly.
---
 tf2_py/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tf2_py/CMakeLists.txt b/tf2_py/CMakeLists.txt
index fd70b0e..7e61ba0 100644
--- a/tf2_py/CMakeLists.txt
+++ b/tf2_py/CMakeLists.txt
@@ -99,6 +99,7 @@ set_target_properties(tf2_py PROPERTIES COMPILE_FLAGS "-g -Wno-missing-field-ini
 set_target_properties(tf2_py PROPERTIES
   ARCHIVE_OUTPUT_DIRECTORY ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_PYTHON_DESTINATION}
   LIBRARY_OUTPUT_DIRECTORY ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_PYTHON_DESTINATION}
+  SKIP_BUILD_RPATH TRUE
 )
 #!! rosbuild_add_compile_flags(tf2_py ${SSE_FLAGS}) #conditionally adds sse flags if available
 
